tigerpath/majors_and_certificates/majors/ECO_2018.json

```
@@ -25,227 +25,261 @@
       "max_counted": 1,
       "min_needed": "ALL",
-      "description": "Prerequisites",
-      "explanation": "To enter the department, a student must complete, by the end of sophomore year, the prerequisite courses ECO 100, ECO 101, ECO 202 (or equivalent), earning a letter grade of C or better in each and a letter grade of C or better in MAT 175. ORF 245 can be substituted for ECO 202; PSY 251, WWS 200 and SOC 301 are not acceptable. The statistics requirement cannot be satisfied with summer courses taken after the student has begun studies at Princeton, except in unusual circumstances approved by the departmental representative, Professor Smita Brunnermeier ",
+      "explanation": "To enter the department, a student must complete, by the end of sophomore year, the prerequisite courses ECO 100, ECO 101, ECO 202 (or equivalent), and MAT (175 or equivalent), earning a letter grade of C or better in each course. ORF 245 can be substituted for ECO 202. POL 345 followed by POL 346 can also be used to satisfy our statistics pre-requirement. (POL 345 alone is not sufficient). PSY 251, WWS 200 and SOC 301 are also not sufficient. The statistics pre-requirement cannot be satisfied with summer courses taken after the student has begun studies at Princeton, except in unusual circumstances approved by the departmental representative, Professor Smita Brunnermeier (smita@princeton.edu).",
       "completed_by_semester": 4,
       "req_list": [
         {
-          "name": "Economics Prerequisites",
-          "max_counted": 2,
-          "min_needed": 2,
-          "description": null,
-          "explanation": "",
-          "course_list": [
-            "ECO 100",
-            "ECO 101"
-          ]
-        },
-        {
-          "name": "Math",
+          "name": "Economics",
           "max_counted": 1,
-          "min_needed": 1,
-          "description": "Math Prerequisite",
-          "explanation": "Students who wish to take ECO 312 or upper-level finance certificate courses (such as ECO 462, ECO 465 or ECO 466), or pursue graduate studies in economics or finance should take the two-semester sequence MAT 201 + MAT 202 (or MAT 203 + MAT 204) rather than MAT 175.",
-          "completed_by_semester": 4,
+          "min_needed": 3,
+          "explanation": "To enter the department, a student must complete, by the end of sophomore year, the prerequisite courses ECO 100, ECO 101, ECO 202 (or equivalent), and MAT (175 or equivalent), earning a letter grade of C or better in each course. ORF 245 can be substituted for ECO 202. POL 345 followed by POL 346 can also be used to satisfy our statistics pre-requirement. (POL 345 alone is not sufficient).",
           "req_list": [
             {
               "name": null,
-              "max_counted": 1,
-              "min_needed": 1,
-              "description": null,
-              "explanation": "",
+              "max_counted": null,
+              "min_needed": "ALL",

```

This just makes the counting more responsive. The logic is the same since the same thing is enforced by the `"min_needed": 3` on line 34.
```suggestion
              "max_counted": 2,
              "min_needed": null,
```